.github/workflows/compile-examples.yml

```diff
@@ -86,7 +86,6 @@ jobs:
               fqbn: "arduino:renesas_uno:unor4wifi"
             additional-sketch-paths: |
               - libraries/WiFiS3
-              - libraries/OTAUpdate
```

Comment:
I assume this is because it breaks the CI build?
I further assume this is broken in `main` and not in your PR? 

If the answer is two times yes, please revert this commit 😉

Reply:
Yes, that's only because `main` is broken by the merge of https://github.com/arduino/ArduinoCore-renesas/pull/100. As long as it tries to compile the OTAUpdate library without a new stable release of this core, it will not find the library :( (bit the `arduino-git` version compiles just fine. I will revert this change at the cost of CI failing.

Reply:
Please revert, I've already created an issue over here ( https://github.com/arduino/ArduinoCore-renesas/issues/121 ) - the CI build failure will be resolved by the next core release.

Reply:
Reverted in 57c7f97c1e9cd88819869cdd72f437907057dd84